mapscript/swiginc/config.i

```diff
@@ -0,0 +1,43 @@
+/* ===========================================================================
+   Project:  MapServer
+   Purpose:  SWIG interface file for mapscript configObj extensions
+   Author:   Steve Lime 
+             Seth Girvin, sgirvin@geographika.co.uk
+             
+   ===========================================================================
+   Copyright (c) 1996-2021 Regents of the University of Minnesota.
+   
+   Permission is hereby granted, free of charge, to any person obtaining a
+   copy of this software and associated documentation files (the "Software"),
+   to deal in the Software without restriction, including without limitation
+   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+   and/or sell copies of the Software, and to permit persons to whom the
+   Software is furnished to do so, subject to the following conditions:
+ 
+   The above copyright notice and this permission notice shall be included
+   in all copies or substantial portions of the Software.
+ 
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+   OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+   DEALINGS IN THE SOFTWARE.
+   ===========================================================================
+*/
+
+%extend configObj 
```

Comment:
@geographika, I just wanted to make sure you agree with the changes I made here:

1. Default for filename is NULL instead of "" so the environment is checked if necessary.
2. I added a deconstructor since the configObj is not freed when the mapObj is freed (and it doesn't even need to be passed to the mapObj constructor.

Reply:
Sounds good. I used an empty string as that was what was in map.i - `mapObj(char *filename="")`